test/run-tests.sh

```diff
@@ -77,4 +77,4 @@ fi
 echo "Running test using number of nodes: ${NUM_NODES}"
 echo "Running test using these images: ${PRIVATE_SPLUNK_OPERATOR_IMAGE} and ${PRIVATE_SPLUNK_ENTERPRISE_IMAGE}..."
 # run Ginkgo
-ginkgo -v -progress -r -stream -nodes=${NUM_NODES} -skipPackage=example,monitoringconsoletest ${topdir}/test -- -commit-hash=${COMMIT_HASH} -operator-image=${PRIVATE_SPLUNK_OPERATOR_IMAGE}  -splunk-image=${PRIVATE_SPLUNK_ENTERPRISE_IMAGE}
+ginkgo -v -progress -r -stream -nodes=${NUM_NODES} --skipPackage=example,ingest_search,monitoring_console,delete_cr ${topdir}/test -- -commit-hash=${COMMIT_HASH} -operator-image=${PRIVATE_SPLUNK_OPERATOR_IMAGE}  -splunk-image=${PRIVATE_SPLUNK_ENTERPRISE_IMAGE}
```

Comment:
Which tests are we actually running as part of this?  Maybe add a comment mentioning what we would like to run here since it's unclear, at least to me.

Reply:
Thats a good idea @jryb. We are only running smoke test cases for PR verification's.